src/components/list/listItem.tsx

```diff
@@ -26,7 +27,7 @@ export default class ListItem extends PureComponent<IItemProps> {
   }
 }
 
-interface IItemProps {
+interface IItemProps extends IItem {
```

Comment:
shouldn't it be IListItemProps extending IListItem

Reply:
right